code/modules/cargo/supplypod_beacon.dm

```diff
@@ -59,6 +61,14 @@
 		express_console.beacon = null
 	return ..()
 
+/obj/item/supplypod_beacon/ex_act()
+	return FALSE
```

Comment:
Wouldn't it be better to give it an armor datum with 100 bomb rotection?

Reply:
fairly sure that armor is meant for humans to not get gibbed by the explosion and not actually reduce bomb damage

Reply:
I've checked the code and it should, but I'll let you know later.

Reply:
was to prevent the explosion from knocking the beacon around, but I'll replace it with 100 bomb armor considering you can anchor it now